charts/opensrp-web/values.yaml

```diff
@@ -88,13 +88,15 @@ containerEnvironmentVariables:
   EXPRESS_OPENSRP_OAUTH_STATE: "{{ .Values.express.opensrpOauthState }}"
   EXPRESS_SESSION_PATH: "{{ .Values.express.sessionPath }}"
   EXPRESS_SESSION_LOGIN_URL: "{{ .Values.express.sessionLoginUrl }}"
-  EXPRESS_FRONTEND_LOGOUT_URL: "{{ .Values.express.sessionFileStorePath }}"
+  EXPRESS_OPENSRP_LOGOUT_URL: "https://{{ .Values.sharedVars.opensrpServer  }}/opensrp/logout.do"
+  EXPRESS_KEYCLOAK_LOGOUT_URL: "https://{{ .Values.sharedVars.keycloakUrl }}/auth/realms/{{ .Values.sharedVars.keycloakRealm }}/protocol/openid-connect/logout"
   EXPRESS_PRELOADED_STATE_FILE: "{{ .Values.express.preloadedStateFile }}"
   EXPRESS_MAXIMUM_LOG_FILES_NUMBER: "5"
   EXPRESS_ALLOW_TOKEN_RENEWAL: "true"
   EXPRESS_MAXIMUM_SESSION_LIFE_TIME: "10800"
-  EXPRESS_LOGS_FILE_PATH: "{{ .Values.express.logsFilePath }}"
-  EXPRESS_SESSION_FILESTORE_PATH: "{{ .Values.express.sessionFileStorePath }}"
+  EXPRESS_LOGS_FILE_PATH: "/tmp/logs/express-server/{{ .Release.Name }}-{{ .Release.NameSpace }}-errors.log"
+  EXPRESS_COMBINED_LOGS_FILE_PATH: "/tmp/logs/express-server/{{ .Release.Name }}-{{ .Release.NameSpace }}-combined.log"
```

Comment:
Any reason why you would want the paths to be unique per deployment?

Reply:
The paths are constant `/tmp/logs/express-server/` what's unique is the file names `{{ .Release.Name }}-{{ .Release.NameSpace }}-combined.log`

Reply:
Okay why should the log file be unique and id `/tmp/` a suitable place for logs?

Reply:
The reasoning was that there could be multiple express deployments in one server (at least in non-k8s deployments) that save log files in same directory thus unique names (eg prod, preview instances) . For the directory, we(the logger) are unable to create sub directories in /var/log because of permission issues. See this https://github.com/onaio/express-server/blob/fbe6d9b25c99817c4924b8e2b79880e1185cef92/src/configs/envs.ts#L93

Reply:
Helm charts are exclusively for k8s deployments. For the permission issue can it be created before hand?

Reply:
*Resolution*:
1. Use the same file-path for logs across all docker deployments
2. The logs should be placed in `var/logs/express/...`, or rather write them to a folder that lives within the code base structure itself. 
  - writing to var is automated within winston and fails due to permissioin errors.
  - Problem is to create the log file in one of the 2 mentioned locations, in a way that is not hindered by permissions.